index.html

```diff
@@ -4,9 +4,41 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+
+<header class="wrapper"> 
+    <h1>Melanie Cascante</h1>  
+
+     <section class="geometrytop">
+        <img src="images/image1.png">
+    </section>
+
+    <div class="toolbar">
+        <nav class="ic_menu">
+            <img src="images/ic_menu.png">
+        </nav>
+    </div>
+</header>
```

Comment:
Good job on using `header` and `nav` tags as requested 💯  but in order to match the design template you picked make sure your hamburger icon is positioned all the way to the right ➡️. See below 👇🏽 

## Your Design
![Screenshot 2023-01-14 at 12 08 09](https://user-images.githubusercontent.com/45739963/212485734-78f06093-5727-438b-8ab8-c41643c00ce0.png)

## Figma Design
![Screenshot 2023-01-14 at 12 08 55](https://user-images.githubusercontent.com/45739963/212485762-c105c592-971a-476e-9e31-64c864f844a3.png)